src/pyskindose/settings_pyskindose.py

```diff
@@ -351,3 +352,16 @@ def create_attrs_str(
             attrs_str = attrs_str + '\n' + getattr(attrs_parent, key).attrs_str
 
     return attrs_str
+
+
+def initialize_settings(settings: Union[str, dict, PyskindoseSettings]) -> PyskindoseSettings:
+    valid_input_settings = settings is not None and isinstance(settings, (str, dict, PyskindoseSettings))
+
+    if not valid_input_settings:
+        raise ValueError("Settings must be given as a str or dict")
+
```

Comment:
Remove superfluous empty line

Reply:
fixed in 09d0a67660f2e07f75547e611b5c52b067246d7e